styles/style.css

```diff
@@ -209,10 +296,40 @@ p {
 
 .facebook-link {
   color: var(--orange-dark);
+  text-align: right;
+}
+
+.fBLink:hover {
+  color: blueviolet;
 }
 
 /* Sidebar */
 
+.sideContainerOne {
+  display:flex;
+  border: 2px solid #e4e4e4; 
+  padding: 20px 10px 15px 15px;
+}
+
+.sideContainerTwo {
+  display:flex;
+  border: 2px solid #e4e4e4;
+  padding: 15px 30px 30px 15px;
+}
+
+                                                      /* added flex boxes to readjust the sidebar*/
+.sideContainerThree {
```

Comment:
Usually in css we try to use the same css rule in one class for similar items, e.g. sidebar items, articles, etc. And then if any of those items need extra css rules, we add a different class for that one. This way we reuse a class, and all the items in our sidebar list looks almost identical in their common features such as `padding`, `border`, etc.